### PR TITLE
Add Detective Pikachu

### PR DESCRIPTION
**Pull request by hernikplays**
Adds Detective Pikachu.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/detective-pikachu/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Detective Pikachu"
+description = "Detective Pikachu (JP) is a Pokémon adventure game by Creatures for the Nintendo 3DS starring a talking Pikachu called Detective Pikachu. The player and Detective Pikachu solve various mysteries in their town."
```
Comment (Hexagon12) [inline games/detective-pikachu/game.dat]: The (JP) is unnecessary?

File: games/detective-pikachu/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
```
Comment (Hexagon12) [inline games/detective-pikachu/game.dat]: Add the respective Title IDs for US and JP. Both were released at 2018-03-23.

File: games/detective-pikachu/game.dat (hunk 1 above)
Comment (Leo626) [reply games/detective-pikachu/game.dat]: I prefer the description from wikipedia 
> Detective Pikachu is an adventure game developed by Creatures Inc., published by The Pokémon Company, and distributed by Nintendo. The game is a spin-off of the Pokémon franchise, in which players work with a talking Pikachu to solve mysteries.

File: games/detective-pikachu/game.dat (hunk 2 above)
Comment (Leo626) [reply games/detective-pikachu/game.dat]: From what I can find, this game uses the `all` region.

**Comment by Leo626** [comment]
Superseded by #298.

**Pull request closed without merging**